code/Code/style.css

```diff
@@ -0,0 +1,151 @@
+/***ALLGEMEIN***/
+@import url("https://fonts.googleapis.com/css2?family=Abel&display=swap");
+
+body {
+  font-family: "Abel", sans-serif;
+  color: black;
+  margin: 0;
+}
+
+/***NAVIGATION***/
+
+.menu-parent {
+  border: 0px solid transparent;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+}
+
+.menu-logo {
+  width: 250px;
+  height: 80px;
+}
+
+.menu-child a {
+  display: inline-block;
+  padding: 10px 15px;
+  margin: 0 5px;
+  text-decoration: none;
+  background: white;
+  color: rgb(5, 5, 5);
+  border-radius: 4px;
+}
+
+.menu-child a:hover {
+  background: rgb(115, 14, 83);
+  color: white;
+}
+
+/***HERO***/
+
+.hero-picture {
+  height: 450px;
+  position: relative;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: rgb(115, 14, 83);
+}
+
+.h1 {
+  font-size: 30pt;
+}
+
+.h2 {
+  font-size: 15pt;
+  color: black;
+}
```

Comment:
Here you could achieve the same thing (having only the `h1` in another color) and save a line of code like this:
```
.hero-text {
  text-align: center;
  position: absolute;
  top: 30%;
  left: 50%;
  transform: translate(-50%, -50%);
}

.h1 {
  font-size: 30pt;
  color: rgb(115, 14, 83);
}

.h2 {
  font-size: 15pt;
}
```